Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora:latest
 # e2fsprogs -- docker @ F20 wants it
```

Comment:
I was wondering about this statement and thought it was left in with a 'purpose'... :-P... removing...